hydrus_transform_control/RollPitchYawGain[] motors
